app/models/work_usage.rb

```diff
@@ -0,0 +1,44 @@
+class WorkUsage
```

Comment:
@gordonleacock is following the pattern already used by Sufia in `app/models/file_usage.rb`. I suggest we hold off on any refactors (of moving model stuff into presenters) until this PR is merged.